content/blog/2020-to-2021.md

```diff
@@ -0,0 +1,49 @@
++++
+title = "2020 -> 2021"
+description = "Reflecting and looking forward."
+date = 2021-01-21
+[extra]
+author = "Josh Lehman"
+ship = "~wolref-podlex"
+image = "https://media.urbit.org/site/posts/essays/arch-sunrise.jpeg"
```

Comment:
Did you take this? I think it's a good rule that images either be either (a) ours or (b) obviously borrowed. Happy to provide an image — will email.

Reply:
I did take this. Was going to save your Tahoe picture for the blog post that you do, assuming you're doing one also.